examples/reference/panes/examples/ipywidgets_rx.py

```diff
@@ -0,0 +1,51 @@
+# Original Source: https://github.com/opengeos/solara-geospatial/blob/main/pages/01_leafmap.py
```

Comment:
This example should be updated and moved to a reference notebook or documentation.